src/modules/identity/RadixRemoteIdentity.ts

```diff
@@ -0,0 +1,108 @@
+import RadixECIES from '../crypto/RadixECIES'
+import RadixIdentity from './RadixIdentity'
+import RadixAccount from '../account/RadixAccount'
+import RadixSignature from '../atom/RadixSignature'
+import RadixSerializer from '../serializer/RadixSerializer'
+
+import { RadixAtom, RadixKeyPair } from '../RadixAtomModel'
+
+export default class RadixRemoteIdentity extends RadixIdentity {
+    private TOKEN: string
+    private REMOTE_URL: string
+
+    constructor(readonly keyPair: RadixKeyPair, token, host = 'localhost', port = '54345') {
+        super(keyPair)
+
+        this.TOKEN = token
+        this.REMOTE_URL = `ws://${host}:${port}`
+    }
+
+    public signAtom(atom: RadixAtom) {
+        return new Promise<RadixAtom>((resolve, reject) => {
+            const _socket = new WebSocket(this.REMOTE_URL)
```

Comment:
Why are you opening a new socket every time in the sign and decrypt methods?

Reply:
Also, the underscore naming scheme, if we were to use it, is typically used for private variables. I know it's still used in a few places, but I think we should move away from it, it's not a recommended practice these days from what I can tell

Reply:
Gotcha, with the `_socket` naming I was mimicking other sockets in the library, and the reason why I open a new socket every time is because we don't know the time between requests (from a developer point of view) to it might get closed. I'll change the naming but I'm not sure about opening just one socket, I think I could ask if the socket is still alive, otherwise open a new one.

Reply:
I think keeping the socket open would be best, but if not, at least close the old ones when you're done using them.